Makefile

```diff
@@ -28,6 +28,12 @@ else
 	BUILDER_IMAGE := quay.io/sustainable_computing_io/kepler_builder:ubi-9-libbpf-1.2.0-go1.18
 endif
 
+ifdef IMAGE_NAME
+	IMAGE_NAME := $(IMAGE_NAME)
+else
+	IMAGE_NAME := kepler
+endif
```

Comment:
How about ... 

```suggestion
IMAGE_NAME ?= kepler
```

Reply:
@sthaha thanks for the suggestion. I refactored the other variable definitions as well for consistency/readability.

Reply:
nice!!